### PR TITLE
Add champ as alias for champion, use JSON highlighting

### PR DESCRIPTION
**Pull request by molenzwiebel**

### CODE REVIEW

File: src/GameData.ts [hunk 1]
```diff
@@ -146,7 +146,8 @@ export default class GameData {
                 result = this.findPerk(searchTerm);
                 break;
             }
-            case "champion": {
+            case "champion": // fall through
+            case "champ": {
```
Comment (stelar7) [inline src/GameData.ts]: isnt there a fancy ES2018 thing we can do to merge the cases? like `case: ["champ", "champion"]`

In other langs you can do things like this, so i assume JS has something similar?
```

   final String numericString = switch (integer)  
      {  
         case 0 -> "zero";  
         case 1, 3, 5, 7, 9 -> "odd";  
         case 2, 4, 6, 8, 10 -> "even";  
         default -> "N/A";  
      };  
```

Comment (molenzwiebel) [reply src/GameData.ts]: Nope. Fallthrough is the best way.

File: src/GameData.ts [hunk 2]
```diff
@@ -157,7 +158,7 @@ export default class GameData {
             return;
         }
 
-        message.channel.send("```" + JSON.stringify(result, null, 4) + "```");
+        message.channel.send("```json\n" + JSON.stringify(result, null, 4) + "```");
```
Comment (stelar7) [inline src/GameData.ts]: message.channel.send("```" + JSON.stringify(result, null, 4) + "```", {code:"json"});

Comment (molenzwiebel) [reply src/GameData.ts]: Then you'd have to leave out the backticks too. `message.channel.send(JSON.stringify(result, null, 4), {code:"json"})`. Both work, but IMO the first one is more clear what is actually sent.

**Comment by Querijn** [comment]
It didn't work, there's an array at the start that was missing the "champ" keyword.

**Pull request merged**